Trappist/src/Promact.Trappist.Core/Controllers/TestConductController.cs

```diff
@@ -1,22 +1,26 @@
-﻿using Microsoft.AspNetCore.Authorization;
+﻿using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
+using Newtonsoft.Json;
 using Promact.Trappist.DomainModel.ApplicationClasses.TestConduct;
+using Promact.Trappist.DomainModel.Enum;
 using Promact.Trappist.DomainModel.Models.TestConduct;
 using Promact.Trappist.Repository.TestConduct;
 using Promact.Trappist.Repository.Tests;
+using System;
+using System.Collections.Generic;
 using System.Threading.Tasks;
-using Newtonsoft.Json;
 
 namespace Promact.Trappist.Core.Controllers
 {
-    [Authorize]
     [Route("api/conduct")]
     public class TestConductController : Controller
     {
         #region Private Variables
         #region Dependencies
         private readonly ITestConductRepository _testConductRepository;
         private readonly ITestsRepository _testRepository;
+        private const string ATTENDEE_ID = "ATTENDEE_ID";
```

Comment:
Ideally, you should use string constants file to define constants which is there in utility.